hail_scripts/shared/elasticsearch_client_v7.py

```diff
@@ -6,6 +6,14 @@
 from pprint import pformat
 
 
+try:
+    import elasticsearch
```

Comment:
you should remove the direct import on line 2 above

Reply:
oops, good call